src/App.jsx

```diff
@@ -1,10 +1,106 @@
-import { useState } from 'react'
-import './App.css'
+import React from 'react';
+import MovieList from './MovieList';
+import Search from './Search';
+import { useEffect, useState } from 'react';
+import Modal from './Modal';
+import './App.css';
+import './Modal.css';
 
-const App = () => {
-  <div className="App">
-    
-  </div>
+function App() {
+  const [movies, setMovies] = useState([]);
+  const [page, setPage] = useState(1);
+  const [searchValue, setSearchValue] = useState('');
+  const [modal, setModal] = useState(false);
+  const [movieID, setMovieID] = useState('');
+
+  const getMovies = async () => {
+    const url = `https://api.themoviedb.org/3/movie/now_playing?language=en-US&page=${page}&api_key=0d7613c1b95dbc61f3dd491c8f802475`;
+    const options = {
+      method: 'GET',
+      headers: {
+        accept: 'application/json',
+        Authorization: 'Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIwZDc2MTNjMWI5NWRiYzYxZjNkZDQ5MWM4ZjgwMjQ3NSIsInN1YiI6IjY2Njc2NmJhODAyN2M0OWNmYjk5ZmJiYyIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.0iAwLYCfoeheb8A8_TWhHNrdDn2P1x3LL-d0_fGs6BA'
+      }
+    };
+
+    const response = await fetch(url, options);
+    const jsonResponse = await response.json();
+    if (page === 1){
+      setMovies(jsonResponse.results)
+    } else {
+      setMovies(prevMovies => [...prevMovies, ...jsonResponse.results])
+    }
+
+  }
+
+  const searchMovies = async () => {
+    const result = searchValue.replace(/ +/g, '+');
+    const url = `https://api.themoviedb.org/3/search/movie?query=${result}&api_key=0d7613c1b95dbc61f3dd491c8f802475`;
+    const options = {
+      method: 'GET',
+      headers: {
+        accept: 'application/json',
+        Authorization: 'Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIwZDc2MTNjMWI5NWRiYzYxZjNkZDQ5MWM4ZjgwMjQ3NSIsInN1YiI6IjY2Njc2NmJhODAyN2M0OWNmYjk5ZmJiYyIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.0iAwLYCfoeheb8A8_TWhHNrdDn2P1x3LL-d0_fGs6BA'
+      }
+    };
+    const response = await fetch(url, options);
+    const jsonResponse = await response.json();
+    const results = jsonResponse.results.filter((movie) => {
+      return movie.title && movie.title.toLowerCase().includes(searchValue.toLowerCase());
+    })
+    if (results.length === 0){
+      return "Try again, No results found :)"
+    }else {
+      setMovies(results);
+    }
+
```

Comment:
Overall looks good to me! The website looks great to me and functioning well! 

Just a few nit on spacing and newlines. I noticed that a lot of functions have a lot of consecutive newlines, shall we delete the newlines to make file structure cleaner?